src/index.html

```diff
@@ -10,8 +11,133 @@
+          <input type="text" name="Surname">
+        </label>
+        <label class="input_type">
+          Name:
+          <input type="text" name="Name">
+        </label>
+        <label class="input_type">
+          How old are You?:
+          <input
+            type="number"
+            name="Age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="input_type">
+          Full date of birth:
+          <input type="date" name="FullDateOfBirth">
+        </label>
+        <label class="input_type">
+          I accept the term of the agreement:
+          <input
+            type="checkbox"
+            name="Agreement"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset_type">
+        <legend>Registration</legend>
+        <label class="input_type">
+          E-mail:
+          <input
+            type="email"
+            name="Email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="input_type">
+          Password:
+          <input
+            type="password"
+            name="Password"
+            minlength="5"
+            maxlength="20"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset_type">
+        <legend>An interesting fact about you!</legend>
+        <div class="input_type">
+          Do you love cats?:
+          <label>
+            <input
+              type="radio"
+              name="DoYouLoveCats?"
+              value="Yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="DoYouLoveCats?"
+              value="No"
+            >
+            No
+          </label>
+        </div>
+        <label class="input_type">
+          What is your favorite color?
+          <input type="color" name="FavoriteColor">
+        </label>
+        <label class="input_type">
+          What time do you go to bed?
+          <input type="time" name="GoToBedAt">
+        </label>
+        <label class="input_type">
+          What are your favorite barand of cars?
+          <select name="FavoriteBrandOfCars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="input_type">
+          How do you rate our work?
+          <input
+            type="range"
+            name="Grade"
+            min="0"
+            max="10"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset_type">
+        <legend>Additional info:</legend>
+        <label class="input_type">
+          Comments:
+          <textarea
+            name="feedback"
+            cols="30"
+            rows="3"
+          >
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="comments"
            cols="30"
            rows="3"
         ></textarea>
```
to avoid this:
![image](https://user-images.githubusercontent.com/86923089/209427325-7e950478-7322-449b-91ca-91243a58dd2b.png)